ui-v2/app/styles/base/components/tooltip/skin.scss

```diff
@@ -7,8 +7,13 @@
   background-color: transparent !important;
   border-left: 9px solid transparent;
   border-right: 9px solid transparent;
+
   border-top: 18px solid $gray-500;
 }
+%tooltip-bottom::after {
+  border-top: 0;
+  border-bottom: 18px solid $gray-500;
```

Comment:
I'm surprised we don't use a variable for more of our standard borders like this.

Reply:
Maybe later we might put some more CSS unit's into variables. Right now we've specifically decided to keep those 'unit based' types of variables light/non-existent.

I think for this particular one, this would only be used in one place. The `18px` dictates the size of the triangle for the tooltips 'tail' using a funny little CSS trick to draw triangles using only CSS, so this `18px` will probably only ever be used in this file for this purpose.